src/tests/suite/daos_test.c

```diff
@@ -295,7 +295,7 @@ main(int argc, char **argv)
 		{"oid_alloc",	no_argument,		NULL,	'O'},
 		{"degraded",	no_argument,		NULL,	'd'},
 		{"rebuild",	no_argument,		NULL,	'r'},
-		{"rebuild_simple",	no_argument,	NULL,	's'},
+		{"rebuild_simple",	no_argument,	NULL,	'v'},
```

Comment:
@kccain v is kind of out of nowhere.   Probably could have picked a letter from word at least like b, perhaps.  Or maybe 'S'

Reply:
Definitely appreciate the comment. In this case I was correcting the code to agree with the other references to -v in the help text and in the run_specified_tests() function. I didn't think to reconsider the choice of letter that had been made previously, perhaps I could have done that and fixed the small number of locations in the file quite easily.